core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -122,6 +122,16 @@ private CarbonCommonConstants() {
    */
   public static final String CARBON_TIMESTAMP_MILLIS = "dd-MM-yyyy HH:mm:ss:SSS";
 
+  /**
+   * CARBON Default TIME format
+   */
+  public static final String CARBON_TIMESTAMP_DEFAULT_FORMAT_TIME = " HH:mm:ss";
```

Comment:
i think this constant name will be confused with timestamp format one, can you update the name in a better way, you can also change the other new ones in this class